src/ValidationError.js

```diff
@@ -20,11 +20,29 @@ class ValidationError extends Error {
     });
 
     this.errors.forEach((err) => {
-      this.message += `options${err.dataPath} ${err.message}\n`;
+      this.message += ValidationError.format(err);
     });
 
     Error.captureStackTrace(this, this.constructor);
   }
+
+  static format(err) {
+    const {
+      keyword,
+      message,
+      dataPath,
+      params: { additionalProperty },
+    } = err;
+    const errorMessage = `options${dataPath} ${message}\n`;
+
+    if (keyword === 'additionalProperties') {
+      return `options${err.dataPath} ${
+        err.message
+      }: ${additionalProperty} is extraneous\n`;
```

Comment:
If we keep the original `Ajv` `err.message` I would prefer to just add the `additionalProperty`

```
options should NOT have additional properties (${additionalProperty})
```